devlab/Vagrantfile

```diff
@@ -46,7 +47,7 @@ Vagrant.configure(2) do |config|
         when "openstack"
           thisnode.vm.provision "shell",
             path: "./provision/keys.sh",
-            args: [ "--public-key", File.read(public_key_path).strip() ]
+            args: [ "--public-key", File.read("#{ENV['HOME']}/#{options['public_key_path'].strip()}").strip() ]
```

Comment:
Once you add `.strip()` above, this `strip()` becomes unneeded and code will be way more readable